app/src/main/res/layout/create_task_item_added.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
I m not sure why do you need create_task_item_added and create_task_item_prompt xml